apps/src/shared/gamma/containers/bottom-tabs/BottomTabsContainer.tsx

```diff
@@ -2,34 +2,34 @@ import React from 'react';
 import type { NativeSyntheticEvent } from 'react-native';
 import {
   BottomTabs,
+  BottomTabsProps,
   BottomTabsScreen,
   BottomTabsScreenProps,
   NativeFocusChangeEvent,
 } from 'react-native-screens';
-import { Colors } from '../../../styling/Colors';
 import ConfigWrapperContext from './ConfigWrapperContext';
 
 export interface TabConfiguration {
   tabScreenProps: BottomTabsScreenProps;
   component: React.ComponentType;
 }
 
-export interface BottomTabsContainerProps {
-  tabConfigs: TabConfiguration[];
-}
+// Currently assumes controlled bottom tabs
+export type BottomTabsContainerProps = Omit<BottomTabsProps, 'experimentalControlNavigationStateInJS' | 'onNativeFocusChange'> & { tabConfigs: TabConfiguration[]; }
 
 export function BottomTabsContainer(props: BottomTabsContainerProps) {
-  // Currently assumes controlled bottom tabs
   console.info('BottomTabsContainer render');
 
+  const { tabConfigs, ...restProps } = props;
```

Comment:
It's ok, don't change it. Just taking opportunity to make sure you're aware here what this syntax does: `...restProps` - AFAIK you create a copy of whole object here.